### ISSUE TITLE
Support for other formatters

### ISSUE THREAD

**Issue by ricardocasares** [skovhus/SublimeLinter-contrib-sass-lint#13]
Maybe it's just me, but when I have a formatter other than `stylish` in my `.sass-lint.yml` Sublime3 does not show the errors/warnings in gutter or anywhere else.

Haven't really checked but it seems that the [parser](https://github.com/skovhus/SublimeLinter-contrib-sass-lint/blob/master/linter.py#L48) cannot find required information when using another report format than `stylish`. If this is the problem, there could be some room for some kind of standardization in the formatters preventing this problem.

Cheers!

**Comment by skovhus** [comment]
Hi @ricardocasares 

Thanks for reporting this. Do you have a concrete example (.sass-lint.yml and a source code) so I can reproduce the issue?

Cheers

**Comment by ricardocasares** [comment]
Sure, in my scss I have just an example function with the wrong indentation (5 spaces instead of 4):

``` sass
@function identity($value) {
     @return $value;
}
```

And this is my `.sass-lint.yml` with the `formatter` option set to `table`

``` yml
files:
  include: 'src/**/*.s+(a|c)ss'
options:
  formatter: table
  merge-default-rules: false
rules:
  indentation:
    - 1
    - size: 4
  nesting-depth:
    - 1
    - max-depth: 4
  no-misspelled-properties:
    - 1
  no-qualifying-elements: 0
  quotes:
    - 1
    - style: single
```

**Comment by skovhus** [comment]
Should be fixed in version 1.0.3.  🎉 

Still waiting for SublimeLinter to update their repo... You can follow along here: https://packagecontrol.io/packages/SublimeLinter-contrib-sass-lint

Please verify @ricardocasares that this solves your problem.

**Comment by ricardocasares** [comment]
@skovhus solved!

Thanks again, cheers!
